tests/testparseperformance.cpp

```diff
@@ -10,6 +10,7 @@
 #include <QFile>
 #include <QDebug>
 #include <QNetworkProxy>
+#include "QTextCodec"
```

Comment:
Qt 6.4 is removing a bunch of unused includes, so you may need to update this commit soon.

Reply:
Qt 6.3 still isn't out. The 6.3 alpha macOS Arm64 binaries the day I started were completely broken and qmake wouldn't find its own mkspec files, so all this above was done with 6.2.3

Reply:
oh, any insights into whether QtLocation is coming back at some point?
Not having the map widget really is a huge let-down :(
QtWebKit is now completely dead and gone and that's a pain - but at least that's only needed for printing and displaying the user manual within the app (that can be worked around using a browser). And who really prints their stuff (I know, I know).
But not having the map widget is really taking a major part of the value away :(

Reply:
It's coming back some time, but I can't tell you when that is. It's still on "status = ignore" in the dev branch today.